config/database.yml.example

```diff
@@ -1,7 +1,6 @@
 default: &default
   adapter: mysql2
-  username: incidents
-  password: password
+  username: root
```

Comment:
We use `root` for round-three and I'm pretty sure that's the only user we have new developers create in their SQL installation during setup.

Reply:
I believe "root" without a password is part of the default mysql installation. I personally use root for all my databases that couldn't have any sensitive data in them. 
But best practices are to have an individual user per database, and I'm not sure if encouraging an easier-but-more-unsafe way is a good thing. Any developer out of training should know how to look in the database.yml, see the user that's listed, and create a user in their mysql installation--or decide to change the user to root.
But then again, setup in this way is a no-brainer and people can indeed decide what they want either way. So those are my thoughts and I'm going to marinate.

Reply:
> I believe "root" without a password is part of the default MySQL installation.

Well yes, but the first thing it does in the setup is ask you to create a password.

It's not really a big deal either way as long as you know how to use MySQL, but I did try to source the SQL file as my incidents user and it failed even though I granted it all privileges (probably my own fault). I just thought it would be easier this way and because it's your local MySQL installation, which shouldn't have any sensitive data on it. So, it shouldn't really matter security wise. I agree It would be better to teach best practices but MySQL does warn you at least three times during setup to use a password so I doubt anyone will forget when it actually matters.

Reply:
The only reason I mentioned that "root without a password is the default" is because you said 
>  that's the only user we have new developers create in their SQL installation during setup.

And I was trying to say we don't _have_ them set up anything. It's just what happens. And yeah, they set up their mysql before working on any repo so no users yet.